web-animations-1/Overview.bs

```diff
@@ -5821,9 +5822,22 @@ dictionary GetAnimationsOptions {
         of the element on which this method was called as the
         <var>timeline</var> argument.
 
-    1.  If <var>options</var> is a {{KeyframeAnimationOptions}} object,
-        assign the value of the <code>id</code> member of <var>options</var> to
-        <var>animation</var>'s {{Animation/id}} attribute.
+    1.  If <var>options</var> is a {{KeyframeAnimationOptions}} object, run the
+        following steps:
+
+        1.  Assign the value of the <code>id</code> member of <var>options</var>
+            to <var>animation</var>'s {{Animation/id}} attribute.
+
+        2.  Let <var>timeline</var> be the <code>timeline</code> member of
+            <var>options</var> or, if <code>timeline</code> member of
+            <var>options</var> is missing, be the
+            <a>default document timeline</a> of the {{Document}}
+            <a lt="document associated with a window">associated</a> with the
+            {{Window}} that is the <a>current global object</a>.
+
+        3.  Run the procedure to <a>set the timeline of an animation</a> on
+            <var>animation</var> passing <var>timeline</var> as the <var>new
+            timeline</var>.
```

Comment:
I wonder if it is observable if an implementation passes the timeline directly to the `Animation` constructor as opposed to setting it after the fact? Maybe it's not currently? If not that's fine, but otherwise we might want to consider what the preferred behavior is.